http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -112,12 +116,22 @@ docker run -d --restart always --network host --name welder \
 --volume "/home/${VM_JUP_USER}":"/work" \
 --env WSM_URL=$WELDER_WSM_URL \
 --env PORT=8081 \
---env WORKSPACE_ID=$WELDER_WORKSPACE_ID \
---env STORAGE_CONTAINER_RESOURCE_ID=$WELDER_STORAGE_CONTAINER_RESOURCE_ID \
+--env WORKSPACE_ID=$WORKSPACE_ID \
+--env STORAGE_CONTAINER_RESOURCE_ID=$WORKSPACE_STORAGE_CONTAINER_ID \
 --env STAGING_STORAGE_CONTAINER_RESOURCE_ID=$WELDER_STAGING_STORAGE_CONTAINER_RESOURCE_ID \
 --env OWNER_EMAIL=$WELDER_OWNER_EMAIL \
 --env CLOUD_PROVIDER="azure" \
 --env LOCKING_ENABLED=false \
 --env STAGING_BUCKET=$WELDER_STAGING_BUCKET \
 --env SHOULD_BACKGROUND_SYNC="false" \
 $WELDER_WELDER_DOCKER_IMAGE
+
+jq --null-input \
```

Comment:
Could you leave a comment explaining that this exposes the desired environment variables to the jupyter kernel?